cpp/tests/sparse/symmetrize.cu

```diff
@@ -214,6 +274,7 @@ const std::vector<COOSymmetrizeInputs<float>> inputsf = {
   }};
 
 INSTANTIATE_TEST_CASE_P(COOSymmetrizeTest, COOSymmetrize, ::testing::ValuesIn(inputsf));
```

Comment:
I think we can remove the old test, as the new test will test the old code-path.